src/client.cpp

```diff
@@ -340,32 +339,12 @@ PVR_ERROR GetChannels(ADDON_HANDLE handle, bool bRadio)
   return PVR_ERROR_SERVER_ERROR;
 }
 
-bool OpenLiveStream(const PVR_CHANNEL &channel)
```

Comment:
It was only used to set a pointless `m_bIsPlaying` variable and `m_currentChannel`, which wasn't really used anywhere either. These methods aren't really used by this addon, they are mainly for addons that handle reading input streams on their own.